api/modules/feedback/views.py

```diff
@@ -30,3 +31,44 @@ def add_feedback(request):
 
     success_message = "Sucessfully added new feedback."
     return Response(success_message, status=status.HTTP_201_CREATED)
+
+
+@api_view(['GET'])
+def get_feedback_id(request, feedback_id):
+    """
+    Returns the feedback pertaining to a certain feedback id
+    :param request:
+    :return: 400 if incorrect parameters are sent or database request failed
+    :return: 201 successful
+    """
+    try:
+        user_feedback = Feedback.objects.get(pk=feedback_id)
+
+    except Feedback.DoesNotExist:
+        error_message = "Feedback doesnt exist"
+        return Response(error_message, status=status.HTTP_404_NOT_FOUND)
+
+    serializer = FeedbackIDSerializer(user_feedback)
+    return Response(serializer.data)
+
+
+""" The many specified here is important because we have a one to many relation and
+has to be specified in both the serializer as well as here. """
+
+
+@api_view(['GET'])
+def get_feedback_all(request):
+    """
+    Returns a list of all the feedbacks for a given user
+    :param request:
+    :param no_of_trips: default 10
```

Comment:
I don't see this parameter being used
Remove this

Reply:
Which parameter?

Reply:
`no_of_trips`